components/Layout/Navigation.js

```diff
@@ -19,6 +19,15 @@ const CurrentUser = (props) => {
   }
 }
 
+const getFilter = () => {
+  if(window.location.pathname ==='/')
+    return null;
+  else if(window.location.pathname === '/options')
+    return <Link className={s.filter} to="/options">Search</Link>
+  else
+    return <Link className={s.filter} to="/options">Filter</Link>
+}
```

Comment:
This is just for the looks. We will have to update the search button to perform search operation in the filter page.